src/server/accessors/OAuthApps.ts

```diff
@@ -0,0 +1,26 @@
+import { IOAuthApp, IOAuthApps } from '../../definition/accessors/IOAuthApps';
+import { OAuthAppsBridge } from '../bridges/OAuthAppsBridge';
+
+export class OAuthApps implements IOAuthApps {
+    constructor(private readonly oauthAppsBridge: OAuthAppsBridge) {}
+
+    public async createOAuthApp(OAuthApp: IOAuthApp, appId: string): Promise<string> {
+        return this.oauthAppsBridge.doCreate(OAuthApp, appId);
+    }
+
+    public async getOAuthAppById(id: string, appId: string): Promise<IOAuthApp> {
+        return this.oauthAppsBridge.doGetByid(id, appId);
+    }
+
+    public async getOAuthAppByName(name: string, appId: string): Promise<Array<IOAuthApp>> {
+        return this.oauthAppsBridge.doGetByName(name, appId);
+    }
+
+    public async updateOAuthApp(OAuthApp: IOAuthApp, id: string, appId: string): Promise<void> {
```

Comment:
Since there is an `id` property in `IOAuthApp`, can we use that instead of passing another `id` as the second paramenter here?

Reply:
The second id refers to the id OAuthApp that will be updated, actually, I need to change the interface of the first parameter to remove the id, since the id is unique and not mutable.